feeds/pl.json

```diff
@@ -134,6 +138,26 @@
             "name": "Szczecin",
             "type": "transitland-atlas",
             "transitland-atlas-id": "f-u36r-zditmszczecin~rt"
+        },
+        {
+            "name": "TransportGZM",
+            "type": "http",
+            "spec": "gtfs",
+            "url": "https://github.com/TransportGZM-GTFS-mirror/TransportGZM-GTFS-normal/releases/latest/download/TransportGZM-GTFS.zip",
+            "license": {
+                "spdx-identifier": "CC-BY-4.0",
+                "url": "https://spdx.org/licenses/CC-BY-4.0.html"
```

Comment:
FYI, this is already the implicit url that get's linked on our sources page, but specifying it doesn't hurt.